colossalai/shardformer/modeling/llama.py

```diff
@@ -390,10 +400,6 @@ def llama_for_sequence_classification_forward(
 
 def get_llama_flash_attention_forward():
 
-    from transformers.models.llama.modeling_llama import LlamaAttention, apply_rotary_pos_emb
-
-    from colossalai.kernel.cuda_native import AttnMaskType, ColoAttention
```

Comment:
Please keep this import here. As flash attn is not always enabled, import it at the top may lead to unexpected import error.

Reply:
done